app/src/test/java/edu/uc/kansakpb/ikeahjalpare3/FurnitureDataUnitTest.kt

```diff
@@ -26,9 +26,9 @@ class FurnitureDataUnitTest {
 
     var furnitureService = mockk<FurnitureService>()
 
-    @Test
+    /*@Test
```

Comment:
The test cases are not brought up tomato to reflect the work that was completed. Not sure what is applicable here and what is not, also should remove mocked objects when you ge the code working. Mocking is for testing specific functions or blocks while building individual elements.

Reply:
Avoid committing commented out code to version control.  This is a no-no, as version control offers better alternatives.
- You're in your own branch, so you can do what you want to do without impacting master.  This gives you a license to experiment.  Instead of commenting out code, delete it.
- You can always go back in history to see what used to be there.

In this specific case, the correct thing to do to a failing unit test is not to remove it, but to fix it.  That's the best course of action for this block.